src/Common/Common.Api/Capacities/Capacity.cs

```diff
@@ -0,0 +1,75 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using Microsoft.PowerBI.Common.Api.Encryption;
+
+namespace Microsoft.PowerBI.Common.Api.Capacities
+{
+    public class Capacity
+    {
+        public Guid Id { get; set; }
+
+        public string DisplayName { get; set; }
+
+        public IEnumerable<string> Admins { get; set; }
+
+        public string Sku { get; set; }
+
+        public CapacityState State { get; set; }
+
+        public CapacityUserAccessRight UserAccessRight { get; set; }
+
+        public string Region { get; set; }
+
+        public Guid? EncryptionKeyId { get; set; }
+
+        public EncryptionKey EncryptionKey { get; set; }
+
+        public static implicit operator Capacity(PowerBI.Api.V2.Models.Capacity capacity)
+        {
+            if (capacity == null)
+            {
+                return null;
+            }
+
+            return new Capacity
+            {
+                Id = Guid.Parse(capacity.Id),
+                DisplayName = capacity.DisplayName,
+                Admins = capacity.Admins,
+                Sku = capacity.Sku,
+                State = (CapacityState)Enum.Parse(typeof(CapacityState), capacity.State, ignoreCase: true),
+                UserAccessRight = (CapacityUserAccessRight)Enum.Parse(
```

Comment:
These values can never be null as per Get API specifications. Reason we do it ConvertCrossFilteringBehavior because these property might be null in some API calls